src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig

```diff
@@ -22,6 +22,9 @@
                 </td>
             {% endif %}
             <td>{{ log.message|format_log_message(log.context) }}</td>
+            {% if log.context ?? false %}
+                <div class="text-muted prewrap log-context">{{ log.context_as_string|raw }}</div>
```

Comment:
currently, you have a XSS issue here: you apply the `raw` filter, but only the rendering using VarDumper is producing HTML properly. Other case are not escaping anything to produce HTML